src/pubkey.h

```diff
@@ -234,7 +240,14 @@ class XOnlyPubKey
 
     const unsigned char& operator[](int pos) const { return *(m_keydata.begin() + pos); }
     const unsigned char* data() const { return m_keydata.begin(); }
-    size_t size() const { return m_keydata.size(); }
+    static constexpr size_t size() { return uint256::size(); }
```

Comment:
in commit b6f6c58ef7e64f699102beef063b3b9f91c71ecd:

Could use `decltype(m_keydata)::size()` for self-documenting code?

Reply:
Done.